test/README.md

```diff
@@ -87,7 +138,111 @@ nohup ./edge_core > edge_core.log 2>&1 &
 
 ### Verify
 
-#### POST request
+**Add Device**
+```bash
+curl -X PUT \
+  http://127.0.0.1:12345/devices \
+  -H 'content-type: application/json' \
+  -d '{
+	"added_devices": [{
+		"id": "kubeedge-device-1",
+		"name": "edgedevice",
+		"description": "integrationtest",
+		"state": "online"
+	}]
+}'
+``` 
+
+#### Verify the DB 
+```bash
+# Enter the database
+sqlite3 edge.db
+
+# Query the database and you shall see the posted Device info
+select * from device;
+```
+
+**Remove Device**
+```bash
+curl -X DELETE \
+  http://127.0.0.1:12345/devices \
+  -H 'content-type: application/json' \
+  -d '{
+	"removed_devices": [{
+		"id": "kubeedge-device-1",
+		"name": "edgedevice",
+		"description": "integrationtest",
+		"state": "online"
+	}]
+}'
+``` 
+
+#### Deploy Application
+```bash
+curl -i -v -X POST http://127.0.0.1:12345/apps -d '{
```

Comment:
maybe we can change the handler in test manager to serve on /pods instead of /apps.
app is a user level concept. pod is what gets deployed.